src/Components/WorkloadsListTable/WorkloadsListTable.js

```diff
@@ -98,8 +140,16 @@ const WorkloadsListTable = ({
             <Link
               to={`${BASE_PATH}/workloads/${item.cluster.uuid}/${item.namespace.uuid}`}
             >
-              <p key={`${index}-cluster`}>{item.cluster.display_name}</p>
-              <p key={`${index}-namespace`}>{item.namespace.name}</p>
+              {item.cluster.display_name ? (
+                <p key={`${index}-cluster`}>{item.cluster.display_name}</p>
+              ) : (
+                <p key={`${index}-cluster`}>{item.cluster.uuid}</p>
+              )}
+              {item.namespace.name ? (
+                <p key={`${index}-cluster`}>{item.namespace.name}</p>
+              ) : (
+                <p key={`${index}-cluster`}>{item.namespace.uuid}</p>
```

Comment:
Typo here, so we can get rid of the same key errors

```suggestion
                <p key={`${index}-namespace`}>{item.namespace.name}</p>
              ) : (
                <p key={`${index}-namespace`}>{item.namespace.uuid}</p>
```